styles/common/_form.scss

```diff
@@ -102,3 +102,66 @@
     border-color: $gray400;
   }
 }
+
+.form-preview-popup {
+  &-content {
+    width: 90% !important; /* stylelint-disable-line declaration-no-important */
+    height: 80%;
+    padding: 0 !important; /* stylelint-disable-line declaration-no-important */
+    border-radius: 5px 5px 0;
+    border: 1px solid #666 !important; /* stylelint-disable-line declaration-no-important */
```

Comment:
why are we using important?

Reply:
We are overriding the popup default properties, we have used important as we can't change those default things cant be changed